core/capabilities/remote/combined_client.go

```diff
@@ -63,9 +85,25 @@ func NewCombinedClient(info capabilities.CapabilityInfo) *combinedClient {
 }
 
 func (c *combinedClient) AddTriggerSubscriber(method string, subscriber capabilities.TriggerCapability) {
```

Comment:
nit: why Add here and not SetMethodTriggerSubscriber/SetMethodExecutableClient ?